roxie/roxiemem/roxiemem.cpp

```diff
@@ -3496,8 +3543,8 @@ class CChunkingRowManager : public CRowManager
     unsigned peakPages;
     unsigned dataBuffs;
     unsigned dataBuffPages;
-    atomic_t possibleGoers;
-    atomic_t totalHeapPages;
+    atomic_uint possibleGoers = {0};
+    atomic_uint totalHeapPages = {0};
```

Comment:
Yes - probably preferred.  These values are used if no initialiser is provided within the constructor.  (Anything used in the constructor takes precedence.)